test/bootstrap.js

```diff
@@ -12,31 +12,10 @@ sinon.assert.expose(assert, { prefix: null });
 // karma-sinon plugins
 globalThis.assert = assert;
 globalThis.sinon = sinon;
+globalThis.context = globalThis.context ?? globalThis.describe;
```

Comment:
Vitest does not have a global `context`, so this simply creates an alias from `describe`, which is basically what with mocha does.